worker/categorize.py

```diff
@@ -403,11 +404,18 @@ def _is_viable_candidate(self, phrase):
                 return False
 
             titled_words = map(lambda x: x.istitle(), words)
-            if titled_words not in [[True, False, True], [True, True,True]]:
+            if titled_words not in [[True, False, True], [True, True, True]]:
                 return False
 
         return True
 
+    def _is_nested(self, keyword, other_keywords):
+        for other_keyword in other_keywords:
+            if keyword != other_keyword and keyword in other_keyword:
```

Comment:
Both "Google Maps" and "Google" will run as the keyword, but we don't need to worry about both being removed because ```"Google Maps" in "Google"``` returns ```false```. This approach will always take the longer option. For example, the Ebola/West Africa test now only returns "Ebola Virus in West Africa" and "Aylward" and "Sierra Leone."